src/workflow/ArcanistPatchWorkflow.php

```diff
@@ -130,9 +130,11 @@ public function getArguments() {
       'uber-merge-using-staging-tag' => array(
         'supports' => array('git'),
         'help' => pht(
-          'Merge from a branch'),
+          'A new branch (git) is created and then '.
+          'the patch is applied, committed and merged in the new branch.'),
```

Comment:
not sure if it should create a new branch. Should just use the existing branch that was pushed to remote as part of arc diff and merge it. (instead of using the created diff)

Reply:
Following our discussion, changed this message